extensions/amp-story/0.1/amp-story.js

```diff
@@ -427,28 +426,39 @@ export class AmpStory extends AMP.BaseElement {
 
   /** @private */
   assertAmpStoryExperiment_() {
-    if (!this.isAmpStoryEnabled_()) {
-      const errorIconEl = this.win.document.createElement('div');
-      errorIconEl.classList.add('i-amphtml-story-experiment-error-icon');
-
-      const errorMsgEl = this.win.document.createElement('span');
-      errorMsgEl.textContent = 'You must enable the amp-story experiment to ' +
-          'view this content.';
-
-      const experimentsLinkEl = this.win.document.createElement('a');
-      experimentsLinkEl.href = `${urls.cdn}/experiments.html`;
-      experimentsLinkEl.textContent = 'Enable or disable your experiments on ' +
-          'the experiments dashboard.';
-
-      const errorEl = this.win.document.createElement('div');
-      errorEl.classList.add('i-amphtml-story-experiment-error');
-      errorEl.appendChild(errorIconEl);
-      errorEl.appendChild(errorMsgEl);
-      errorEl.appendChild(experimentsLinkEl);
-      this.element.appendChild(errorEl);
-
-      user().error(TAG, 'enable amp-story experiment');
+    if (this.isAmpStoryEnabled_()) {
+      return;
     }
+
+    const errorIconEl = this.win.document.createElement('div');
+    errorIconEl.classList.add('i-amphtml-story-experiment-error-icon');
+
+    const errorMsgEl = this.win.document.createElement('span');
+    errorMsgEl.textContent = 'You must enable the amp-story experiment to ' +
+        'view this content.';
+
+    const experimentsLinkEl = this.win.document.createElement('button');
+    experimentsLinkEl.textContent = 'Enable';
+    experimentsLinkEl.addEventListener('click', () => {
+      this.enableAmpStoryExperiment_();
+    });
+
+    const errorEl = this.win.document.createElement('div');
+    errorEl.classList.add('i-amphtml-story-experiment-error');
+    errorEl.appendChild(errorIconEl);
+    errorEl.appendChild(errorMsgEl);
+    errorEl.appendChild(experimentsLinkEl);
+    this.element.appendChild(errorEl);
+
+    user().error(TAG, 'enable amp-story experiment');
+  }
+
+
+  /** @private */
+  enableAmpStoryExperiment_() {
+    toggleExperiment(this.win, 'amp-story', true);
+    toggleExperiment(this.win, 'dev-channel', true);
```

Comment:
Why is `dev-channel` necessary?

Reply:
It's not strictly necessary, but since this is an experimental component, folks generally want the more bleeding edge rather than the robustness.  I don't feel strongly about this; I'm open to removing it.

Reply:
FYI I believe this is a no-op on pub origins. The AMP Cache checks its own cookie before deciding which version of `v0.js` to serve.

Reply:
Removed the `dev-channel`